packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -142,6 +146,24 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 		return toDeltaManagerInternal(this.runtime.deltaManager);
 	}
 
+	private setUpTelemetryEventBatcher(): TelemetryEventBatcher<keyof ITelemetryProperties> {
+		assert(
+			this.mc !== undefined && this.logger !== undefined,
+			0x349a /* this.mc and/or this.logger has not been set */,
```

Comment:
Curious if there's a better practice when creating a new error code with identical error message.

Reply:
We should actually never copy an assert code from somewhere else, in this case just go back to the string message that's commented. The assert-tagging infrastructure will eventually put a new code in here before the next release. Asserts codes should uniquely identify a place in the code where they came from, so having the same one in two places would be an issue.